src/components/Footer.astro

```diff
@@ -31,6 +32,10 @@ import { Icon } from "astro-icon";
             <a href="/support" title="Support Vencord's Development">
                 <Icon name="fa-regular:heart" />
             </a>
+            <Theme client:load>
+                <Icon name="fa-regular:sun" slot="light"/>
```

Comment:
these Icons should have a title indicating their functionality! 
```suggestion
                <Icon name="fa-regular:sun" slot="light" title="Switch to dark theme" />
```

Reply:
Thanks for pointing that out, I completely forgot that this (and it also made me realize that it was always showing the current theme's icon and not the one it was going to switch to)